File: Spring_Back/src/main/java/com/busanit401/spring_back/dto/DeliveryOrderDTO.java
package com.busanit401.spring_back.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class DeliveryOrderDTO {
    private Long orderId;
    private Long userId;
    private int totalAmount;
    private String deliveryAddress;
    private String status;
    private LocalDateTime createdAt;
}

File: flutter_front/lib/domain/dto/delivery_order_dto.dart
class DeliveryOrderDto {
  final String status;

  DeliveryOrderDto({required this.status});

  // 백엔드가 읽을 수 있도록 자바 Object(Map) 형태로 변환해주는 함수
  Map<String, dynamic> toJson() {
    return {
      'status': status,
    };
  }
}